ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -765,3 +765,9 @@ def remainder(self, y, name=None):
 
     def is_floating_point(self):
         return paddle_frontend.is_floating_point(self._ivy_array)
+
+    @with_supported_dtypes(
+        {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle"
+    )
+    def unstack(self, axis=0, name=None):
```

Comment:
The function takes another argument `num`. You could refer to the docs [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/unstack_en.html).

Reply:
@bipinKrishnan Done